src/db.rs

```diff
@@ -2007,6 +2007,45 @@ impl<T: ThreadMode, D: DBInner> DBCommon<T, D> {
         }
     }
 
+    pub fn get_column_family_metadata(&self) -> Result<ColumnFamilyMetaData, Error> {
```

Comment:
A couple of comments:
1. It would be good to add some description to the public methods.
2. What's the point of the `Result` here?
3. I guess it makes sense to check the pointer `ptr` for null.

Reply:
1. Done. 
2. I was thinking that it's better for design because C-API can change at any time. Removed.
3. As I can see from the code, it cannot be null.